src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -157,7 +165,7 @@ const PersonalDetails = (() => {
             displayChangeableFields(data);
             $(real_acc_elements).setVisibility(1);
             showHideTaxMessage();
-            CommonFunctions.getElementById('tax_information_form').setVisibility(shouldShowTax(get_settings));
+            CommonFunctions.getElementById('tax_information_form').setVisibility(1);
```

Comment:
if we want to make it visible for all real accounts, should we handle it as part of the `real_acc_elements` then?

Reply:
Done, I added the RealAcc class to this element in personal_details and removed this line of code.